conf/messages.en

```diff
@@ -147,3 +147,41 @@ taxableProfit.chargeableProfitsOnCTR = Chargeable profits for Corporation Tax on
 taxableProfit.whereCanIFindMyCompanyTaxProfit = Where can I find my company''s taxable profit?
 taxableProfit.youCanFindYourCompanyProfitVia = You can find your company's profit via your accountant or accounting software
 taxableProfit.hint = Must be entered as a whole number. For example, 70000
+
+distribution.title = Did your company receive any distributions?
```

Comment:
move this to the distribution section above

Reply:
see above